coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs',
+    versions = 1.0, 1.1, 1.2, 1.3, 1.4, 98, 2010
+    special_chars = list('::!#$%&;*+.,/<=>?@\\^_|-~')
```

Comment:
Doesn't `special_chars` means it is a list of char so `::` is equal to `:`?
@jayvdb please help :joy: